The Game of Luck 1.2 (no sensors).py

```diff
@@ -0,0 +1,126 @@
+### This is not the smart version. It cannot detect any input error.
+
+import random as r
+
+#let's define all our variables here
+list0 = []; list1 = ["heads", "tails"]; list2 = ["rock", "paper", "scissors"]; list3 = ["a", "b", "c"]; list4 = []
+for i in range(5):
+    list4.append(i + 1)
+# lists
+
+tot1 = 0; tot2 = 0; tot3 = 0; tot4 = 0; tot5 = 0   #totals
+ 
+b = 0; s = 0; l = 0   #integers
+c = 0; n = 0; v = 0   #strings
+
+#now, let's define all our needed functions, or collections of code, here:
+
+def add(tot5):
```

Comment:
Tot 5 is outside the scope of the add function. If the variable is above a function, it's considered outside its scope.
Try fixing by moving the variables around